pkg/kubectl/cmd/wait/wait.go

```diff
@@ -315,24 +322,25 @@ func IsDeleted(info *resource.Info, o *WaitOptions) (runtime.Object, bool, error
 			}
 			return gottenObj, false, errWaitTimeoutWithName
 		default:
+			watchErr, ok := err.(*watchtools.WatcherError)
+			if ok {
+				fmt.Fprintf(w.ErrOut, "error: An error occurred while waiting for the condition to be satisfied: %v", watchErr)
```

Comment:
Related to my last question, why do we need w.ErrOut, why can't we use o.ErrOut?